paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/NewPaymentMethodRowButton.kt

```diff
@@ -64,7 +66,7 @@ internal fun NewPaymentMethodRowButton(
                 iconUrl = iconUrl,
                 imageLoader = imageLoader,
                 iconRequiresTinting = iconRequiresTinting,
-                modifier = modifier.size(20.dp),
+                modifier = modifier.height(20.dp).width(24.dp),
```

Comment:
Should we make these constants so we reuse them between rows?

Reply:
do you mean to share them between the new + saved payment method row buttons? If so, yeah I can do that

Reply:
updated!